cpp/modmesh/buffer/SimpleCollector.hpp

```diff
@@ -115,6 +115,20 @@ class SimpleCollector
         return SimpleArray<T>(expander().as_concrete());
     }
 
+    void push_back(value_type const & value)
```

Comment:
`SimpleCollector::push_back` uses the same interface as `std::vector::push_back`.